src/store/modules/nodes/nodes-plugin.js

```diff
@@ -15,14 +15,16 @@ export default (store) => {
     const { type, payload } = mutation
 
     if (type === 'nodes/useFastest') {
-      nodes.adm.setUseFastest(!!mutation.payload)
+      const selectedNodeType = payload.type()
```

Comment:
payload.type is not a function, it's a string

```suggestion
      const selectedNodeType = payload.type
```